src/libs/actions/Policy.ts

```diff
@@ -657,6 +672,26 @@ function setWorkspacePayer(policyID: string, reimburserEmail: string, reimburser
     API.write(WRITE_COMMANDS.SET_WORKSPACE_PAYER, params, {optimisticData, failureData, successData});
 }
 
+function clearWorkspaceAutoReportingError(policyID: string) {
+    Onyx.merge(`${ONYXKEYS.COLLECTION.POLICY}${policyID}`, {errorFields: {autoReporting: null}});
+}
+
+function clearWorkspaceAutoReportingFrequencyError(policyID: string) {
+    Onyx.merge(`${ONYXKEYS.COLLECTION.POLICY}${policyID}`, {errorFields: {autoReportingFrequency: null}});
+}
+
+function clearWorkspaceGeneralSettingError(policyID: string) {
+    Onyx.merge(`${ONYXKEYS.COLLECTION.POLICY}${policyID}`, {errorFields: {generalSettings: null}});
+}
+
+function clearWorkspaceAutoReportingOffsetError(policyID: string) {
+    Onyx.merge(`${ONYXKEYS.COLLECTION.POLICY}${policyID}`, {errorFields: {autoReportingOffset: null}});
+}
+
+function clearWorkspaceApprovalError(policyID: string) {
+    Onyx.merge(`${ONYXKEYS.COLLECTION.POLICY}${policyID}`, {errorFields: {approvalMode: null}});
+}
+
```

Comment:
I wonder if we can DRY this since they all clear a policy errorField. Maybe something like:

```
function clearPolicyErrorField(fieldName: string) {
Onyx.merge(`${ONYXKEYS.COLLECTION.POLICY}${policyID}`, {errorFields: {[fieldName]: null}});
} 
```